[INFO]2022-07-03 11:30:06,778: #################################################################
[INFO]2022-07-03 11:30:06,779: Perform log
[INFO]2022-07-03 11:30:06,780: #################################################################
[INFO]2022-07-03 11:30:06,783: name:56:Use case writer:zpou
[INFO]2022-07-03 11:30:06,785: Enter function : flow_weather()
[INFO]2022-07-03 11:30:06,785: Input parameter : update_dict, value : value is empty.
[INFO]2022-07-03 11:30:06,786: Enter function : flow_login()
[INFO]2022-07-03 11:30:06,787: Input parameter : update_dict, value : value is empty.
[INFO]2022-07-03 11:30:06,788: apileft:56:interface message: {'url': '/api/mgr/signin', 'type': 'post', 'api_type': 'http', 'location_interface': 'my_api_login', 'request': {'username': 'byhy', 'password': '88888888'}, 'reponses_type': 'json', 'Content-Type': 'data'}
[INFO]2022-07-03 11:30:06,791: apileft:65:requests url: http://127.0.0.1:80/api/mgr/signin
[INFO]2022-07-03 11:30:06,792: apileft:69:request method: post
[INFO]2022-07-03 11:30:06,793: apileft:74:requests parameter: {'username': 'byhy', 'password': '88888888'}
[INFO]2022-07-03 11:30:06,794: apileft:80:headers: {'content-type': 'application/x-www-form-urlencoded; charset=UTF-8', 'x-requested-with': 'XMLHttpRequest', 'sec-ch-ua-platform': 'Windows', 'Sec-Fetch-Site': 'same-origin', 'Sec-Fetch-Mode': 'cors', 'Sec-Fetch-Dest': 'empty', 'Cookie': 'csrftoken = ftE3dWoJqtQx963qTSGEtPPD62a3dntwBQWBacLeot1E1ntBWX7DE75O6EhU53kg'}
[DEBUG]2022-07-03 11:30:06,798: Starting new HTTP connection (1): 127.0.0.1:80
[DEBUG]2022-07-03 11:30:07,343: http://127.0.0.1:80 "POST /api/mgr/signin HTTP/1.1" 200 10
[INFO]2022-07-03 11:30:07,344: respones:96:In response to the results: {'ret': 0}
[INFO]2022-07-03 11:30:07,346: flow_weather:29:TypeError: update_dict is not dict!
[ERROR]2022-07-03 11:30:07,347: apileft:82:name 'false' is not defined
[ERROR]2022-07-03 11:30:07,349: apileft:83:Traceback (most recent call last):
  File "D:\my_api\tc_play\requetes.py", line 55, in apileft
    contex = eval(context)
  File "<string>", line 16, in <module>
NameError: name 'false' is not defined

[ERROR]2022-07-03 11:30:07,350: post:61:local variable 'parm' referenced before assignment
[ERROR]2022-07-03 11:30:07,351: post:62:Traceback (most recent call last):
  File "D:\my_api\Public_variable\apitest.py", line 50, in post
    param = self.Requests.apileft(api, dict_request)
  File "D:\my_api\tc_play\requetes.py", line 85, in apileft
    return parm[0], parm[1], parm[2], url, headers, request_method
UnboundLocalError: local variable 'parm' referenced before assignment

[ERROR]2022-07-03 11:30:07,353: flow_weather:35:local variable 'respones_reslut' referenced before assignment
[ERROR]2022-07-03 11:30:07,354: flow_weather:36:Traceback (most recent call last):
  File "D:\my_api\confdb\flow\user\client.py", line 30, in flow_weather
    weather_result = self.api_get_apone(weather_dict)
  File "D:\my_api\confdb\api\user\client.py", line 19, in api_get_apone
    return self.post(Tim_api_mgr_customers, request_dict)
  File "D:\my_api\Public_variable\apitest.py", line 64, in post
    return respones_reslut
UnboundLocalError: local variable 'respones_reslut' referenced before assignment

